images/win/toolsets/toolset-2022.json

```diff
@@ -308,7 +308,7 @@
     },
     "mysql": {
         "major_version": "8.0",
-        "full_version": "8.0.26"
+        "full_version": "8.0.27"
```

Comment:
Don't update mysql version for Windows Server 2022.

Reply:
@nikolai-frolov what @al-cheb said :) I was downgrading 8.0.27 to 8.0.26 but picked 5.7 not on purpose too (see https://github.com/actions/virtual-environments-internal/issues/2999)